Misc/NEWS.d/next/Library/2024-11-27-23-29-05.gh-issue-109798.OPj1CT.rst

```diff
@@ -0,0 +1,2 @@
+Normalized error messages for both implementations of :mod:`datetime`. Made
+more detailed error messages showing input values. Patch by Semyon Moroz.
```

Comment:
I think we can shorten this. No need to explicitly mention that it was both implementations.
```suggestion
Normalized error messages for :mod:`datetime`, and made
them more useful when debugging. Patch by Semyon Moroz.
```